.github/workflows/release.yml

```
@@ -53,5 +54,10 @@ jobs:
           git config --global user.name "semantic-release"
           git config --global user.email "semantic-release@users.noreply.github.com"
           poetry run semantic-release version
-          poetry run semantic-release publish
+          poetry build
 
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@ed0c53931b1dc9bd32cbe73a98c7f6766f8a527e
+        if: steps.semantic-release.outcome == 'success'
```

actionlint (1.7.7)

property "semantic-release" is not defined in object type {generate_token: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)